src/view.ts

```diff
@@ -0,0 +1,45 @@
+import { Frame } from "./types";
+
+/**
+ * The view function takes an array of Frame objects as
+ * its argument.
+ *
+ * You can see the explicit structure of a Frame object
+ * in the `types.ts` file, and examples of Frame objects
+ * in the `frames.ts` file.
+ *
+ * The view function loops through the array of Frames,
+ * and for each one it builds and inserts the following:
+ *
+ * Create a `div` element (we'll call it `container`)
+ * which will serve as the container for all of the
+ * Frame's elements. Using `container.appendChild`,
+ * we'll insert the following elements into the
+ * container.
+ *
+ *   - An h2 element, into which we'll insert the Frame's
+ *     `action` text.
+ *
+ *   - A `ul` element, to contain the options the user
+ *     has.
+ *
+ *   - The view function loops through the `options`
```

Comment:
Is this part of the list of elements? If so, maybe start with the name of the element ('li') and then explain it to keep the list consistent?